railcard.css

```
@@ -38,18 +40,29 @@
 }
 #railcard #railcard_headbar .rarity5,
 #railcard #railcard_headbar .rarity4 {
-    width: 56px;
-    box-shadow: 2px 2px 2px 1px whitesmoke;
+    width: 100%;
+    box-shadow: 2px 2px 5px 1px;
     border-width: 2px 2px 0px 2px;
     border-style: double;
     border-color: rgba(255, 255, 255, 0.5);
 }
+
+#railcard_detail_img:hover {
+  transition: transform 0.3s ease-in-out;
+  -webkit-transform: scale(0.2);
+  -ms-transform: scale(0.2);
+  transform: scale(1.1);
+}
+
 #railcard #railcard_detail {
     display: flex;
     align-items: center;
     justify-content: center;
     width: 100%;
 }
 #railcard #railcard_detail_img {
-    max-width: 100%;
-}
\ No newline at end of file
+    transition: transform 0.3s ease-in-out;
+    max-width: 90%;
+    border-radius: 30px;
```

the border radius is also a bit large, some of your card (like template 3) could be clipped improperly.

```suggestion
    border-radius: 10px;
```